ballerina/http_client.bal

```diff
@@ -127,40 +125,40 @@ client isolated class HttpClient {
         return externSubmit(self, httpVerb, path, <Request>message);
     }
 
-    # Retrieves the `http:Response` for a previously-submitted request.
+    # Get the response from a previously submitted asynchronous request. Can be used after calling `submit()` action to
+    # retrieve the actual response.
     #
     # + httpFuture - The `http:HttpFuture` related to a previous asynchronous invocation
     # + return - An `http:Response` message or else an `http:ClientError` if the invocation fails
     remote isolated function getResponse(HttpFuture httpFuture) returns Response|ClientError {
         return externGetResponse(self, httpFuture);
     }
 
-    # Checks whether an `http:PushPromise` exists for a previously-submitted request.
+    # Check if the server has sent a push promise for additional resources. Can be used with HTTP/2 server push functionality.
```

Comment:
This is not supported for HTTP/1.0 or HTTP/1.1. So `Can be used with HTTP/2` might be misleading. May be we should use `Should be used with HTTP/2`

Reply:
```suggestion
    # Check if the server has sent a push promise for additional resources. Should be used with HTTP/2 server push functionality.
```